### PR TITLE
Add the Amazon EC2 Depolyment docs

### PR DESCRIPTION
**Pull request by raman08**
Screenshot from 2021-01-16 15-52-24
Screenshot from 2021-01-16 15-52-36
Screenshot from 2021-01-16 15-52-48

### CODE REVIEW

**Comment by raman08** [comment]
For Issue: #31 

@smaranjitghose,  Please tell me if this formate is right so that I can make the other docs in the same formate

File: docs/hosting.md [hunk 1]
```diff
@@ -161,3 +161,123 @@ kubectl get service
...
+
+- Now, clone your repository into instance.
+
+```console
+ git clone link_to_your_repository.git
+```
+
+- The next step is to  `cd `  into repository and install required packages using
+
+```console
```
Comment (smaranjitghose) [inline docs/hosting.md]: Giving the console keyword for code block at times hinders conversion so I would advise against using it

File: docs/hosting.md [hunk 2]
```diff
@@ -161,3 +161,123 @@ kubectl get service
 *** Will be covered later
 
 
+## Amazon EC2
+
+- Visit [AWS Portal](https://aws.amazon.com/free/).
+
+- Log in.
+If you dont have an account set it up. It will ask you for **Credit Card details**.  <br><br>
```
Comment (smaranjitghose) [inline docs/hosting.md:169]: Do not use line break <br>. At times markdown conversion breaks due to this

File: docs/hosting.md [hunk 3]
```diff
@@ -161,3 +161,123 @@ kubectl get service
...
+- Follow [this]( https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/putty.html?icmpid=docs_ec2_console).
+
+Now we are in our AWS EC2 Instance.
+
+#### Setting Up Environment
+
+- The next step is to install python and all the necessary libraries for your app.
+
+```console
+sudo yum install python3 git
```
Comment (smaranjitghose) [inline docs/hosting.md]: I guess yum is used primarily for Red Hat. Try to go for an approach using the GitHub repo itself

File: docs/hosting.md [hunk 4]
```diff
@@ -161,3 +161,123 @@ kubectl get service
...
+- Now, clone your repository into instance.
+
+```console
+ git clone link_to_your_repository.git
+```
+
+- The next step is to  `cd `  into repository and install required packages using
+
+```console
+python3 -m pip install streamlit
```
Comment (smaranjitghose) [inline docs/hosting.md:259]: These are to be done from requirements.txt or using the docker container

File: docs/hosting.md [hunk 5]
```diff
@@ -161,3 +161,123 @@ kubectl get service
...
+```
+
+- Now, just run the app and you can see your app live on the External URL.
+
+```console
+streamlit run app.py
```
Comment (smaranjitghose) [inline docs/hosting.md:265]: Preferred using Docker

**Review by smaranjitghose — APPROVED** [review]

**Comment by smaranjitghose** [comment]
If you do not have experience with Cloud Hosting ignore my above comments except the ones related to markdown syntax. Rest looks fine to me

File: docs/hosting.md (hunk 1 above)
Comment (raman08) [reply docs/hosting.md]: Okk sir

File: docs/hosting.md (hunk 2 above)
Comment (raman08) [reply docs/hosting.md:169]: Okk sir

File: docs/hosting.md (hunk 3 above)
Comment (raman08) [reply docs/hosting.md]: Sir, should I change it to apt?

File: docs/hosting.md (hunk 4 above)
Comment (raman08) [reply docs/hosting.md:259]: Sorry sir, I did not know about docker.

File: docs/hosting.md (hunk 3 above)
Comment (smaranjitghose) [reply docs/hosting.md]: ya

**Comment by smaranjitghose** [comment]
@raman08 Updates?

**Pull request merged**